native/acter/src/api/verification.rs

```diff
@@ -1077,11 +1113,11 @@ impl SessionManager {
 }
 
 impl Client {
-    pub fn verification_event_rx(&self) -> Option<Receiver<VerificationEvent>> {
-        match self.verification_controller.event_rx.try_lock() {
-            Ok(mut r) => r.take(),
-            Err(e) => None,
-        }
+    // this return value should be unpinned when wait_for_verification_event ends, because it calls pin_mut
+    // this return value should be wrapped in Box::pin, to make unpin possible
```

Comment:
```suggestion
```

not necessary doc comments

Reply:
Frankly, I don't want to forget this logic.
If I leave this comment, I can remember that later.